app/assets/javascripts/angular/controllers/repos_ctrl.js

```diff
@@ -4,42 +4,86 @@ app.controller("ReposCtrl", ["GitSearch", "$scope", function(GitSearch, $scope)
   $scope.size = [];
   $scope.stars = [];
   $scope.forks = [];
+  $scope.commitHistory = [];
 
-  $scope.searchRepo = function(user, repo) {
-    GitSearch.searchRepo(user, repo).success(function(data){
+  $scope.searchRepo = function(user,repo) {
+    GitSearch.searchRepo(user,repo).success(function(data){
 
-        var newByte = convertBytes(data.size)
-
-        $scope.forks.push({key: data.name, y: data.forks});
-        $scope.issues.push({key: data.name, y: data.open_issues});
-        $scope.size.push({key: data.name, y: data.size});
-        $scope.stars.push({key: data.name, y: data.stargazers_count});
+      var newByte = convertBytes(data.size)
 
+      $scope.forks.push({key: data.name, y: data.forks});
+      $scope.issues.push({key: data.name, y: data.open_issues});
+      $scope.size.push({key: data.name, y: data.size});
+      $scope.stars.push({key: data.name, y: data.stargazers_count});
     });
-  }
+  };
+
+  $scope.searchCommits = function(user,repo) {
+    GitSearch.searchCommits(user,repo).success(function(data){
+
+      var obj = [];
+      for ( var i in data) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.